config.ru

```diff
@@ -3,10 +3,12 @@ require 'bundler/setup'
 $:.unshift File.expand_path('..', __FILE__)
 require 'app/controllers/app_controller'
 
-require 'rack/attack'
-require_relative 'config/rack_attack'
+if ENV['RACK_ENV'] != 'development'
+  require 'rack/attack'
+  require_relative 'config/rack_attack'
 
-use Rack::Attack
+  use Rack::Attack
+end
```

Comment:
You need a lot of ENV vars set up to do development, where this isn't particularly relevant.